src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -128,11 +159,16 @@ private function parseFile(string $filePath) : void
         foreach ($this->cachingVisitor->getFunctionNodes() as $functionName => $functionNode) {
             $this->functionNodes[$functionName] = $functionNode;
         }
+
+        /** @psalm-suppress UndefinedMethod */
+        foreach ($this->cachingVisitor->getConstantNodes() as $constantName => $constantNode) {
```

Comment:
this wouldn't result in an `UndefinedMethod` if the underlying anonymous class was a dedicated class (and same for the FunctionReflector